io/ilvis2/Ilvis2Reader.cpp

```diff
@@ -123,11 +125,83 @@ T convert(const StringList& s, const std::string& name, size_t fieldno)
 }
 
 
+// If longitude between 0-180, just return it, degrees east; if between 180
+// and 360, subtract 360 to get negative value.
+double Ilvis2Reader::convertLongitude(double longitude)
+{
+  if (longitude > 180.0)
+  {
+    return longitude - 360.0;
+  }
+  else
+  {
+    return longitude;
+  }
+}
+
+
+void Ilvis2Reader::readPoint(PointViewPtr view, PointId nextId, StringList s, std::string pointMap)
+{
+  PointLayoutPtr layout = view->layout();
+
+  std::string name("LVIS_LFID");
```

Comment:
Up to you, but I'll just say that I was going to make this change anyway and probably will at some point if you don't.  Just because they're in the list doesn't mean they're general.  It means that we know about them from the specification -- they aren't variable strings that we don't know until runtime.  I allowed for 65,000+ known dimensions.  We have a long way to go :)